stable/database/README.md

```diff
@@ -307,6 +307,19 @@ The following tables list the configurable parameters for the `database` option
 | ----- | ----------- | ------ |
 | `nuodb.config` | [NuoDB database options][6] | `nil` |
 
+#### restore.*
```

Comment:
In this one re are also missing doco on the autoImport and autoRestore options

Reply:
Yup - good catch.

README updated - will be included in the next push.